sound/soc/intel/boards/Kconfig

```diff
@@ -466,4 +467,17 @@ config SND_SOC_INTEL_TGL_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE
 
+config SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH
+        tristate "CML with RT1011 and RT5682 in I2S Mode"
+        depends on MFD_INTEL_LPSS && I2C && ACPI
```

Comment:
I've already mentioned you need to align with other machine drivers and add COMPILE_TEST.